src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -32,19 +29,24 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
+import org.bukkit.Chunk;
+import org.bukkit.Location;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Coordinate")
-@Description("Represents a given coordinate of a location. ")
-@Examples({"player's y-coordinate is smaller than 40:",
-		"	message \"Watch out for lava!\""})
-@Since("1.4.3")
-public class ExprCoordinate extends SimplePropertyExpression<Location, Number> {
+@Description("Represents a given coordinate of a location/chunk. ")
+@Examples({
+	"player's y-coordinate is smaller than 40:",
+	"\tmessage \"Watch out for lava!\"",
+	"player's chunk's z-coordinate is bigger than 10",
+	"\tmessage \"You're leaving the underground!\""
+})
+@Since("1.4.3, INSERT VERSION (chunks)")
+public class ExprCoordinate extends SimplePropertyExpression<Object, Number> {
 	
 	static {
-		register(ExprCoordinate.class, Number.class, "(0¦x|1¦y|2¦z)(-| )(coord[inate]|pos[ition]|loc[ation])[s]", "locations");
+		register(ExprCoordinate.class, Number.class, "(0¦x|1¦y|2¦z)(-| )(coord[inate]|pos[ition]|loc[ation])[s]", "locations/chunks");
```

Comment:
These should be using the new parse tags.

Reply:
Please don't mark a RC conversation as resolved until a commit is pushed RE the change.

Reply:
```suggestion
		register(ExprCoordinate.class, Number.class, "(0:x|1:y|2:z)(-| )(coord[inate]|pos[ition]|loc[ation])[s]", "locations/chunks");
```